resources/assets/enderio/lang/de_DE.lang

```diff
@@ -530,6 +976,18 @@ enderio.gui.travelAccessable.protected=Geschützt
 enderio.gui.travelAccessable.unauthorised=Dir ist es nicht erlaubt, auf diesen Block zuzugreifen.
 enderio.gui.travelAccessable.enterCode=Zugriffscode benötigt
 enderio.gui.travelAccessable.ok=Autorisieren
+enderio.gui.travelAccessable.skipLocked=Überspringe abgeschlossenen Anker, benutzte gezielten Teleport.
+enderio.gui.travelAccessable.skipObstructed=Überspringe versperrten Anker.
+enderio.gui.travelAccessable.skipPrivate=Überspringe privaten Anker.
+
+enderio.gui.telepad.configure.travel=Reiseeinstellungen
+enderio.gui.telepad.configure.telepad=Telepad-Einstellungen
+enderio.gui.telepad.blocked=Teleport blockiert!
+
+//Spawn Gurad [sic]
+enderio.gui.spawnGurad.showRange=Zeige Reichweite
+enderio.gui.spawnGurad.hideRange=Verstecke Reichweite
```

Comment:
"Verberge" (Verstecken is only "hide" in the sense of putting something in a secret place where you don't want to to be found, where verbergen is taking something out of sight)